src/_includes/amazon-intelligent-pricing-ceiling-configure.md

```diff
@@ -6,12 +6,12 @@ Select your optional highest price setting in the **Optional Ceiling Price** sec
 
 1. For **Ceiling Price Action**, select an option.
 
-    - **Decrease By**: Choose when you want the defined ceiling price source value to be decreased before listing to Amazon.
+    - **Decrease By**: Choose when you want the defined Ceiling Price Source value to be adjusted down, creating a lower ceiling price for the rule, before listing to Amazon.
```

Comment:
I thought we decided on `-` insteand of `:`

Reply:
for Amazon, everything was done using :   that would be a global change for amazon docs.

Reply:
OK -- then we should leave for now. At some point we should change them for consistency.